source/designer/source/DocumentCanvas.cpp

```diff
@@ -67,7 +67,7 @@ void DocumentCanvas::SelectControls( ControlList& CtrlList )
 {
 	m_selectionLayer->ClearSelection();
 
-	for ( ControlList::List::const_iterator it = CtrlList.list.begin(); it != CtrlList.list.end(); ++it )
+	for ( ControlList::List::const_iterator it = CtrlList.list.cbegin(); it != CtrlList.list.cend(); ++it )
```

Comment:
If you have a look at the GWEN code, which is pre-C++11, you'll see it has changed quite a lot already. GWEN "works" but there are quite a lot of improvements that can be made. TBH it might have been faster to start from scratch!

Changing these makes the code look nicer, but isn't essential. If you'd like to do it then that would be great.